infrastructure/cdn-in-a-box/dns/Dockerfile

```diff
@@ -23,10 +23,10 @@ ENV BIND_USER=bind \
 RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
  && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends \
  && apt-get update \
- && DEBIAN_FRONTEND=noninteractive apt-get install -y vim.tiny wget net-tools sudo net-tools ca-certificates unzip apt-transport-https \
+ && DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends install -y vim.tiny wget net-tools sudo net-tools ca-certificates unzip apt-transport-https \
  && rm -rf /var/lib/apt/lists/* && rm -rf /etc/apt/apt.conf.d/docker-gzip-indexes \
  && apt-get update \
- && DEBIAN_FRONTEND=noninteractive apt-get install -y bind9=${BIND_VERSION}* bind9-host=${BIND_VERSION}* dnsutils \
+ && DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends install -y bind9=${BIND_VERSION}* bind9-host=${BIND_VERSION}* dnsutils \
```

Comment:
Same as above—We already disable `APT::Install-Recommends` in the APT configuration in this Dockerfile, so no need to pass the `--no-install-recommends` CLI option.